optaplanner-core/src/test/java/org/optaplanner/core/api/score/buildin/bendable/BendableScoreTest.java

```diff
@@ -205,7 +205,8 @@ public void feasibleHHSSS() {
         assertScoreFeasible(
                 scoreDefinitionHHSSS.createScoreInitialized(0, 0, -300, -4000, -5000),
                 scoreDefinitionHHSSS.createScoreInitialized(0, 2, -300, -4000, -5000),
-                scoreDefinitionHHSSS.createScoreInitialized(2, 0, -300, -4000, -5000)
+                scoreDefinitionHHSSS.createScoreInitialized(2, 0, -300, -4000, -5000),
+                scoreDefinitionHHSSS.createScoreInitialized(1, -20, -300, -4000, -5000)
```

Comment:
This is a bug in optaplanner-core. I 've created a jira and I am now fixing it:
  https://issues.jboss.org/browse/PLANNER-716

scoreDefinitionHHSSS.createScoreInitialized(1, -20, -300, -4000, -5000) should be infeasible due to the -20.